flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskExecutor.java

```diff
@@ -404,8 +404,6 @@ private void stopTaskExecutorServices() throws Exception {
 			exception = ExceptionUtils.firstOrSuppressed(e, exception);
 		}
 
-		taskSlotTable.stop();
```

Comment:
where is this now called?

Reply:
It's always been also part of `taskExecutorServices.shutDown();`, further below